rmf_site_format/src/workcell.rs

```diff
@@ -280,6 +282,59 @@ pub struct Workcell {
     // TODO(luca) Joints
 }
 
+impl From<Pose> for urdf_rs::Pose {
+    fn from(pose: Pose) -> Self {
+        urdf_rs::Pose {
+            rpy: match pose.rot {
+                Rotation::EulerExtrinsicXYZ(arr) => urdf_rs::Vec3(arr.map(|v| v.radians().into())),
+                Rotation::Yaw(v) => urdf_rs::Vec3([0.0, 0.0, v.radians().into()]),
+                Rotation::Quat([x, y, z, w]) => {
+                    let (x, y, z) = glam::quat(x, y, z, w).to_euler(EulerRot::XYZ);
```

Comment:
I actually don't know but I noticed that in the matching function in [misc.rs](https://github.com/open-rmf/rmf_site/blob/d2fb61f8785e66021b56286fb64b6fda454f9ffe/rmf_site_format/src/misc.rs#L266-L267) we actually apply the rotation in ZYX, not sure what should be done but just thought I'd point out the difference.

Reply:
I switched to use zyx in c8ef823.